Files/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -21,6 +21,7 @@
                     x:Name="chkPermanentlyDelete"
                     x:Uid="ConfirmDeleteDialogPermanentlyDeleteCheckBox"
                     x:Load="{x:Bind ViewModel.PermanentlyDeleteLoad, Mode=OneWay}"
+                    BorderThickness="0"
```

Comment:
Why the change here?

Reply:
I've copied the code from somewhere else, I'll remove it.